log_max_archives 내용 수정 및 추가 (#687)
CUBRID/cubrid-manual: en/admin/config.rst

Reviewer: ```suggestion
    *   Required memory size = the size of log buffer (**log_buffer_size**\)
```
해당 이슈와 관계 없으나 **) 를 **슬래시)로 권고합니다.

Author: 리뷰 의견 감사합니다. 반영하였습니다.

**log_max_archives**

>     **log_max_archives** is a parameter to configure the maximum number of archive log files. The minimum value is 0 and default value is **INT_MAX** (2,147,483,647). Its operations can differ depending on the configuration of **force_remove_log_archives**. For example, when **log_max_archives** is 3 and **force_remove_log_archives** is **yes** in the cubrid.conf file, the most recent three archive log files are recorded and when a fourth archiving log file is generated, the oldest archive log file is automatically deleted; the information about the deleted archive logs are recorded in the ***_lginf** file. If the user sets ha_mode=off and force_remove_log_archives=n, then the log_max_archives value is set to INT_MAX (2,147,483,647).

Reviewer 2: ```suggestion
    **log_max_archives** is a parameter to configure the maximum number of archive log files. The minimum value is 0 and default value is **INT_MAX** (2,147,483,647). Its operations can differ depending on the configuration of **force_remove_log_archives**. For example, when **log_max_archives** is 3 and **force_remove_log_archives** is **yes** in the cubrid.conf file, the most recent three archive log files are recorded and when a fourth archiving log file is generated, the oldest archive log file is automatically deleted; the information about the deleted archive logs are recorded in the ***_lginf** file. When a user sets ha_mode=off and force_remove_log_archives=n, the system automatically assigns the value of INT_MAX (2,147,483,647) to log_max_archives instead of the value entered by the user.
```

Author: 수정 후 반영하였습니다. 리뷰 감사합니다.

>     **log_max_archives** is a parameter to configure the maximum number of archive log files. The minimum value is 0 and default value is **INT_MAX** (2,147,483,647). Its operations can differ depending on the configuration of **force_remove_log_archives**. For example, when **log_max_archives** is 3 and **force_remove_log_archives** is **yes** in the cubrid.conf file, the most recent three archive log files are recorded and when a fourth archiving log file is generated, the oldest archive log file is automatically deleted; the information about the deleted archive logs are recorded in the ***_lginf** file. When a user sets ha_mode=off and force_remove_log_archives=n, system automatically assigns the value of INT_MAX (2,147,483,647) to log_max_archives instead of the value entered by the user.

Reviewer 3: ```suggestion
    **log_max_archives** is a parameter to configure the maximum number of archive log files. The minimum value is 0 and default value is **INT_MAX** (2,147,483,647). Its operations can differ depending on the configuration of **force_remove_log_archives**. For example, when **log_max_archives** is 3 and **force_remove_log_archives** is **yes** in the cubrid.conf file, the most recent three archive log files are recorded and when a fourth archiving log file is generated, the oldest archive log file is automatically deleted; the information about the deleted archive logs are recorded in the ***_lginf** file. When a user sets ha_mode=off and force_remove_log_archives=n, the system automatically assigns the value of INT_MAX (2,147,483,647) to log_max_archives instead of the value entered by the user.
```

Author: 리뷰 감사합니다. 수정하였습니다.
